webpack/webpack.dev.js

```diff
@@ -0,0 +1,17 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common');
+
+module.exports = merge(common, {
+  mode: 'development',
+
+  output: {
+    filename: '[name].js',
+    chunkFilename: '[name].chunk.js',
+    publicPath: '/',
+  },
+
```

Comment:
`devtool` and `devServer` could be for debugging. Check this out:
https://webpack.js.org/configuration/devtool/

My previous project's configs: 

```javascript
devtool: 'eval-source-map',
	devServer: {
		stats: {
			colors: true,
			hash: false,
			version: false,
			timings: true,
			assets: false,
			chunks: false,
			modules: false,
			reasons: false,
			children: false,
			source: true,
			errors: true,
			errorDetails: false,
			warnings: false,
			publicPath: false
		},
		// server default port
		port: 3001,
		historyApiFallback: true
	},
```

Reply:
I will add it when config webpack for development